client/src/components/CatalogMainSection/CatalogMainSection.module.scss

```diff
@@ -0,0 +1,21 @@
+$mobile: 428 px;
+$tablet: 768px;
+$laptop: 1024px;
+$desktop: 1360px;
+
+$primaryMainColor: #0499dd;
+
+.mainSection {
+  background: center url('./catalogMainSectionBG.jpg') no-repeat;
```

Comment:
I guess it`s better to move image to the following path : client/public/assets/images/ Your folder here / Your Image here.

@Ahmed-Fahmy-Dev , please comment if I`m right here with the file structure.

Reply:
absolutely write.   @AlinaPalyvoda  please act accordingly.

Reply:
Done